lib/auth.js

```diff
@@ -105,42 +105,48 @@ export default class Auth {
     constructor(authObject, options) {
         if (!authObject) {
             throw new Error('authObject are required. see readme.');
-        } else if (!authObject.client_id) {
-            throw new Error('client_id or client_secret is missing or invalid');
-        } else if (typeof authObject.client_id !== 'string') {
-            throw new TypeError('client_id or client_secret must be strings');
-        } else if (!authObject.client_secret) {
-            throw new Error('client_id or client_secret is missing or invalid');
-        } else if (typeof authObject.client_secret !== 'string') {
-            throw new TypeError('client_id or client_secret must be strings');
-        } else if (!authObject.account_id) {
-            throw new Error('account_id is missing or invalid');
-        } else if (!Number.isInteger(Number.parseInt(authObject.account_id))) {
-            throw new TypeError(
-                'account_id must be an Integer (Integers in String format are accepted)',
-            );
-        } else if (!authObject.auth_url) {
-            throw new Error('auth_url is missing or invalid');
-        } else if (typeof authObject.auth_url !== 'string') {
-            throw new TypeError('auth_url must be a string');
-        } else if (
-            !/https:\/\/[\da-z-]{28}\.auth\.marketingcloudapis\.com\//gm.test(authObject.auth_url)
-        ) {
-            throw new Error(
-                'auth_url must be in format https://mcXXXXXXXXXXXXXXXXXXXXXXXXXX.auth.marketingcloudapis.com/',
-            );
-        } else if (authObject.scope && !Array.isArray(authObject.scope)) {
-            throw new Error('Scope must be in array format');
-        } else if (authObject.scope && getInvalidScopes(authObject.scope).length > 0) {
-            throw new Error(
-                getInvalidScopes(authObject.scope)
-                    .map((value) => '"' + value + '"')
-                    .join(',') + ' is/are invalid scope(s)',
-            );
+        } else if (!authObject.access_token) {
```

Comment:
to improve readability can you do something like below? 

```suggestion
        } else if (authObject.access_token) {
           // no action needed
        }
        
```

Reply:
Fixed it.